docs/reference/ilm/apis/remove-policy.asciidoc

```diff
@@ -1,21 +1,23 @@
 [role="xpack"]
 [testenv="basic"]
-[[ilm-remove-policy]]
-=== Remove Policy On Index API
+[[ilm-delete-policy]]
+=== Delete Policy from Index API
 ++++
-<titleabbrev>Remove Policy From Index</titleabbrev>
+<titleabbrev>Delete Policy</titleabbrev>
 ++++
 
-Unassigns a policy from a specified index pattern
+Removes the assigned lifecycle policy from an index.
 
 ==== Request
 
 `DELETE <index>/_ilm`
 
 ==== Description
 
-This action removes a policy from managing an index. It is effectively the same as setting an index's
-`index.lifecycle.name` setting to null.
+Removes the assigned lifecycle policy and stops managing the specified index.
+If an index pattern is specified, removes the assigned policies from all matching
+indices. This has the same effect as setting an index's `index.lifecycle.name`
+setting to `null`.
```

Comment:
```
This has the same effect as setting an index's `index.lifecycle.name`
setting to `null`.
```
We should remove this sentence as its not true since this API also cleans up the other ILM state in teh index metadata.

Reply:
Fixed.